bentoml/server/bento_api_server.py

```diff
@@ -41,17 +40,20 @@
 
 INDEX_HTML = '''\
 <!DOCTYPE html>
-<head><link rel="stylesheet" type="text/css"
-            href="/static/swagger-ui.css"></head>
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="static/swagger-ui.css">
+    <title>Swagger</title>
+</head>
 <body>
-<div id="swagger-ui-container"></div>
-<script src="/static/swagger-ui-bundle.js"></script>
-<script>
-    SwaggerUIBundle({{
-        url: '{url}',
-        dom_id: '#swagger-ui-container'
-    }})
-</script>
+    <div id="swagger-ui-container"></div>
+    <script src="https://unpkg.com/swagger-ui-dist@3/swagger-ui-bundle.js"></script>
```

Comment:
One benefit of not using CDN is that the user can use this web UI even without the internet. A few early users requested this a while back so we changed it including "swagger-ui-bundle.js" in /static and use it instead

Reply:
Yeah, I was having syntax errors with the local version of the script, not intended to be a permanent change (that's why I didn't change the style sheet)